src/Platform_Core/org/lobobrowser/context/AlgorithmDigest.java

```diff
@@ -0,0 +1,131 @@
+package org.lobobrowser.context;
+
+import java.security.MessageDigest;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class AlgorithmDigest implements Comparable<AlgorithmDigest> {
+
+  private final String algorithm;
+  private final String digest;
+  private final Integer strength;
+  private final static Map<String, Integer> str = createMap();
+
+  public AlgorithmDigest(String algorithm, String digest, Integer strength) {
+    super();
+    this.algorithm = algorithm;
+    this.digest = digest;
+    this.strength = strength;
+  }
+
+  public static boolean validate(final byte[] input, final String integrity) {
+
+    final List<AlgorithmDigest> algDigests = parseMetadata(integrity);
+    System.out.println("Strongest algorithms: " + algDigests);
+
+    System.out.println("The value of the integrity attribute is: " + integrity);
+
+    if (algDigests == null)
+      return true;
```

Comment:
code style: surround with braces.

For a classic example of the problem that braces protect against, see: https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/